### PR TITLE
design: improve responsive display

### LINKED ISSUE #365: Sign in / Sign up distinction
The visibility of the "Sign in" button is not satisfying.
Suggestions:
* either like Explore: "My account" button with mousehover showing possibility to sign in in big, and possibility to sign up in small (1 click on PC)
* either like RERO ILS: "My account" menu with the two possibilities below (2 clicks)
* either "Sign in" button in evidence and in small below a link "New here ? Register"

### LINKED ISSUE #474: Improve responsive display

### LINKED ISSUE #476: Update frontpage
- [ ] Update the presentation text and increase font size and width.
- [ ] Remove `Project website on sonar.ch` in `Follow us` section.
- [ ] Update links at the bottom with the following: `Help & Documentation | IR hosting service | About SONAR | Contact`.

**Presentation text:**
The Swiss Open Access Repository is an archive of scholarly publications. It collects, promotes and preserves open access publications by authors affiliated with Swiss public research institutions. It does so by aggregating content and metadata from international sources as well as from institutional repositories (IR) of Swiss Higher Education Institutions (HEI).

### PR DESCRIPTION
**Pull request by sebastiendeleze**
* Improves the display in small screens.
* Changes the links and display for sign in and sign up actions.
* Changes occurrences `Log in` for `Sign in`.
* Updates the frontpage.
* Closes #365.
* Closes #474.
* Closes #476.

Co-Authored-by: Sébastien Délèze <sebastien.deleze@rero.ch>

### CODE REVIEW

**Review by pronguen — COMMENTED** [review]
* Footer: the target of https://about.sonar.ch/contact/ doesn't exist.
* Not sure that "Sign up" should be more visible than "Sign in", the latter is more frequently used.

File: sonar/theme/templates/sonar/accounts/forgot_password.html [hunk 1]
```diff
@@ -42,12 +42,12 @@ <h4 class="text-center mb-4">{{_('Reset Password')}}</h3>
       </div>
       {%- if current_user.is_anonymous %}
         <div class="mt-2 text-right">
-            <a href="{{url_for('security.login')}}" class="text-light mr-3">{{_('Log In')}}</a>
+            <a href="{{url_for('security.login')}}" class="text-light mr-3">{{_('Log in')}}</a>
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/accounts/forgot_password.html:45]: space after `{` and before `}`

File: sonar/theme/templates/sonar/accounts/forgot_password.html [hunk 2]
```diff
@@ -42,12 +42,12 @@ <h4 class="text-center mb-4">{{_('Reset Password')}}</h3>
...
             {%- if security.registerable %}
-              <a href="{{url_for('security.register')}}" class="text-light">{{_('Sign Up')}}</a>
+              <a href="{{url_for('security.register')}}" class="text-light">{{_('Sign up')}}</a>
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/accounts/forgot_password.html:47]: space after `{` and before `}`

File: sonar/theme/templates/sonar/accounts/login.html [hunk 3]
```diff
@@ -30,20 +30,20 @@ <h4 class="text-center mb-4">{{_('Log in to account') }}</h3>
           {{form_errors(form)}}
           {{ render_field(form.email, autofocus=True, errormsg=False) }}
           {{ render_field(form.password, errormsg=False) }}
-          <button type="submit" class="btn btn-secondary btn-lg btn-block text-light"><i class="fa fa-sign-in"></i> {{_('Log In')}}</button>
+          <button type="submit" class="btn btn-secondary btn-lg btn-block text-light"><i class="fa fa-sign-in"></i> {{_('Log in')}}</button>
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/accounts/login.html:33]: space after `{` and before `}`

File: sonar/theme/templates/sonar/accounts/login.html [hunk 4]
```diff
@@ -53,7 +53,7 @@ <h4 class="text-center mb-4">{{_('Log in to account') }}</h3>
     <div class="mt-2 text-right">
         <a href="{{url_for('security.forgot_password')}}" class="text-light mr-3">{{_('Forgot password?')}}</a>
         {%- if security.registerable %}
-          <a href="{{url_for('security.register')}}" class="text-light">{{_('Sign Up')}}</a>
+          <a href="{{url_for('security.register')}}" class="text-light">{{_('Sign up')}}</a>
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/accounts/login.html:56]: space after `{` and before `}`

File: sonar/theme/templates/sonar/accounts/reset_password.html [hunk 5]
```diff
@@ -34,9 +34,9 @@ <h4 class="text-center mb-4">{{_('Reset Password')}}</h3>
       </div>
       {%- if current_user.is_anonymous %}
         <div class="mt-2 text-right">
-            <a href="{{url_for('security.login')}}" class="text-light mr-3">{{_('Log In')}}</a>
+            <a href="{{url_for('security.login')}}" class="text-light mr-3">{{_('Log in')}}</a>
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/accounts/reset_password.html:37]: space after `{` and before `}`

File: sonar/theme/templates/sonar/accounts/reset_password.html [hunk 6]
```diff
@@ -34,9 +34,9 @@ <h4 class="text-center mb-4">{{_('Reset Password')}}</h3>
...
             {%- if security.registerable %}
-              <a href="{{url_for('security.register')}}" class="text-light">{{_('Sign Up')}}</a>
+              <a href="{{url_for('security.register')}}" class="text-light">{{_('Sign up')}}</a>
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/accounts/reset_password.html:39]: space after `{` and before `}`

File: sonar/theme/templates/sonar/frontpage.html [hunk 7]
```diff
@@ -21,42 +21,46 @@
...
+    <div class="row justify-content-center">
+      <div class="col-8 col-xs-6 col-lg-3 py-3 py-sm-5 text-center">
+        <a
+          href="{{ url_for('documents.index', view=view_code if g.get('organisation', {}).get('isDedicated') else config.SONAR_APP_DEFAULT_ORGANISATION) }}">
+          {% if g.get('organisation', {}).get('isDedicated') %}
+          {% set thumbnail = g.organisation | record_image_url %}
+          {% if thumbnail %}
+          <img src="{{ thumbnail }}" class="img-fluid">
+          {% else %}
+          <h1>{{g.organisation.name}}</h1>
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/frontpage.html:34]: space after `{` and before `}`

File: sonar/theme/templates/sonar/frontpage.html [hunk 8]
```diff
@@ -21,42 +21,46 @@
...
+    </div>
+    <div class="row justify-content-center">
+      <div class="col-lg-8 text-right my-4">
+        <form class="justify-content-end"
+          action="{{ url_for('documents.search', view=view_code if g.get('organisation', {}).get('isDedicated') else config.SONAR_APP_DEFAULT_ORGANISATION) }}"
+          role="search">
+          <div class="row">
+            <div class="col-9 col-md-11">
+              <input class="form-control form-control-lg mr-2" type="search"
+                placeholder="{{_('Search publications, authors, projects...')}}" aria-label="Search" name="q">
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/frontpage.html:50]: space after `{` and before `}`

File: sonar/theme/templates/sonar/frontpage.html [hunk 9]
```diff
@@ -69,32 +73,27 @@ <h6 class="m-0 mt-1">
...
 {%- block body %}
-  <div class="row justify-content-between">
-    <div class="col-12 col-lg-5 pr-5">
-      <h3 class="mb-4">{{_('The project')}}</h2>
-      <p>{{_('The SONAR project aims to create a scholarly archive that collects, promotes and preserves the publications of authors affiliated with Swiss public research institutions.')}}</p>
-      <p>{{_('Further info on the project page')}} <a href="https://sonar.ch/project" target="_blank">sonar.ch</a>.</p>
+<div class="row justify-content-between">
+  <div class="col-lg-7">
+    <h5 class="home-text">
+      {{_('The Swiss Open Access Repository is an archive of scholarly publications. It collects and promotes open access publications by authors affiliated with Swiss public research institutions.')}}
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/frontpage.html:79]: space after `{` and before `}`

File: sonar/theme/templates/sonar/frontpage.html [hunk 10]
```diff
@@ -69,32 +73,27 @@ <h6 class="m-0 mt-1">
...
+    </h5>
+  </div>
+  <div class="col-lg-4 mt-3 mt-lg-0">
+    <h3 class="mb-4">{{_('Follow us')}}</h3>
+    <div class="row align-items-center mt-3">
+      <div class="col-2">
+        <i class="fa fa-github fa-2x"></i>
+      </div>
+      <div class="col-10">{{_('Source code on')}} <a href="https://github.com/rero/sonar" target="_blank">Github</a>
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/frontpage.html:88]: space after `{` and before `}`

File: sonar/theme/templates/sonar/frontpage.html [hunk 11]
```diff
@@ -69,32 +73,27 @@ <h6 class="m-0 mt-1">
...
-      <p>{{_('The SONAR project aims to create a scholarly archive that collects, promotes and preserves the publications of authors affiliated with Swiss public research institutions.')}}</p>
-      <p>{{_('Further info on the project page')}} <a href="https://sonar.ch/project" target="_blank">sonar.ch</a>.</p>
+<div class="row justify-content-between">
+  <div class="col-lg-7">
+    <h5 class="home-text">
+      {{_('The Swiss Open Access Repository is an archive of scholarly publications. It collects and promotes open access publications by authors affiliated with Swiss public research institutions.')}}
+    </h5>
+  </div>
+  <div class="col-lg-4 mt-3 mt-lg-0">
+    <h3 class="mb-4">{{_('Follow us')}}</h3>
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/frontpage.html:83]: space after `{` and before `}`

File: sonar/theme/templates/sonar/partial/navbar.html [hunk 12]
```diff
@@ -1,82 +1,77 @@
...
-    <a href="{{url_for_security('login', next=request.path)}}" class="btn btn-light ml-auto mr-2 d-block d-lg-none">
-      <i class="fa fa-lock"></i>
+    {% if page != 'home' %}
+    <a class="navbar-brand" href="{{ url_for('documents.index', view=view_code) }}">
+      {% if g.get('organisation', {}).get('isDedicated') %}
+      {% set thumbnail = g.organisation | record_image_url %}
+      {% if thumbnail %}
+      <img src="{{ thumbnail }}" height="50" class="d-inline-block align-top mr-3 my-2" alt="">
+      {% else %}
+      {{g.organisation.name}}
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/partial/navbar.html:27]: space after `{` and before `}`

File: sonar/theme/templates/sonar/partial/navbar.html [hunk 13]
```diff
@@ -1,82 +1,77 @@
...
-      <form action="{{ url_for('documents.search', view=view_code if g.get('organisation', {}).get('isDedicated') else config.SONAR_APP_DEFAULT_ORGANISATION) }}"
+      {% if page != 'home' %}
+      <form
+        action="{{ url_for('documents.search', view=view_code if g.get('organisation', {}).get('isDedicated') else config.SONAR_APP_DEFAULT_ORGANISATION) }}"
         class="form-inline my-2 my-lg-0 ml-lg-3">
-        <input name="q" class="form-control {{ 'form-control-sm' if g.get('organisation') else '' }} mr-sm-2" type="search" placeholder="{{_('Search')}}"
-          aria-label="{{_('Search')}}" value="{{ request.args.get('q', '') }}">
-        <button class="btn btn-outline-light {{ 'btn-sm' if g.get('organisation') else '' }} my-2 my-sm-0" type="submit">
+        <input name="q" class="form-control {{ 'form-control-sm' if g.get('organisation') else '' }} mr-sm-2"
+          type="search" placeholder="{{_('Search')}}" aria-label="{{_('Search')}}"
```
Comment (Garfield-fr) [inline sonar/theme/templates/sonar/partial/navbar.html:48]: space after `{` and before `}`

**Review by Garfield-fr — APPROVED** [review]

**Comment by sebastiendeleze** [comment]
> * Footer: the target of https://about.sonar.ch/contact/ doesn't exist.

This page will be created by @mmo. It already exists in the preview website https://60230e5cadadb33bfce52a8c--sonardotch.netlify.app/contact/.

> * Not sure that "Sign up" should be more visible than "Log in", the latter is more frequently used. I am open to other opinion, but this does not cover any of the proposals in #365

I think the call to action for `Register` should be kept highlighted. People who already have a login, know where the link is. Those who don't have one, should quickly find the button `Register`. This is the case on most applications.

**Review by lauren-d — APPROVED** [review]

**Pull request merged**